src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -341,12 +338,12 @@ function enableNewQuestion() {
  * @param questionNum
  */
 function disableQuestion(questionNum) {
-    var $currentQuestionTable = $('#questionTable' + questionNum);
+    var $currentQuestionTable = $('#questionTable-' + questionNum);
 
     $currentQuestionTable.find('text,button,textarea,select,input').prop('disabled', true);
     
-    $currentQuestionTable.find('#mcqAddOptionLink').hide();
-    $currentQuestionTable.find('#msqAddOptionLink').hide();
+    $currentQuestionTable.find('[id^="mcqAddOptionLink-"]').hide();
+    $currentQuestionTable.find('[id^="msqAddOptionLink-"]').hide();
```

Comment:
Leaving a trailing hyphen here doesn't really seem necessary, but I guess it's okay to leave it as it is, since any attempts to remove the hyphen from the original id would cause this to break, alerting the dev that something's off.